templates/fine-tune-llm_v2/cookbooks/bring_your_own_data/README.ipynb

```diff
@@ -0,0 +1,89 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Bring your own data \n",
+    "**⏱️ Time to complete**: 10 minutes\n",
+    "\n",
+    "This guide focuses on how you can bring your own data to fine-tune your model on the Anyscale Platform. Make sure you've gone over the [basic fine-tuning guide](../../README.md) before going over this cookbook.\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data stored in remote storage \n",
+    "\n",
+    "## Public datasets\n",
+    "For datasets configured for public access, you simply need to add the relevant training and validation file URI in your training YAML. We support loading from data stored on S3 and GCS.\n",
```

Comment:
show some snippet YAML files and how it would look like for train / validation.

Reply:
Showed an example training YAML.